app/classifier/feedback/feedback-summary-container.jsx

```diff
@@ -35,24 +38,30 @@ class FeedbackSummaryContainer extends React.Component {
     }, []);
     actions.feedback.setFeedback(feedbackItems);
   }
+
+  render() {
+    return (this.props.feedback.length) ? <FeedbackSummary feedback={this.props.feedback} /> : null;
```

Comment:
No I mean using `this.props.feedback.length > 0` as @eatyourgreens suggests since that is an explicit statement that doesn't rely on implicit conversion. Javascript can have unintuitive truthy/falsy behavior, so we shouldn't rely on implicit conversion when possible.